### PR TITLE
Add atok

### PR DESCRIPTION
**Pull request by ohkura**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [*] `brew cask audit --download {{cask_file}}` is error-free.
- [*] `brew cask style --fix {{cask_file}}` reports no offenses.
- [*] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [*] Named the cask according to the [token reference].
- [*] `brew cask install {{cask_file}}` worked successfully.
- [*] `brew cask uninstall {{cask_file}}` worked successfully.
- [*] Checked there are no [open pull requests] for the same cask.
- [*] Checked the cask was not already refused in [closed issues].
- [*] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/atok.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  version '2016'
+  sha256 'bda513644013a35c352624f74b4b918f35650e9a88155ecc758d978cddb1a9ae'
+
+  url 'https://gate.justsystems.com/download/atok/ut/mac/at29try.dmg'
+  name 'ATOK'
+  homepage 'http://www.justsystems.com/jp/products/atokmac/'
+
+  pkg 'ATOK 2016 インストーラ.pkg'
+
+  # uninstall script: '/Applications/JustSystems/ATOK\ 2016/ATOK\ Uninstaller.app/Contents/MacOS/ATOK\ Uninstaller'
```
Comment (miccal) [inline Casks/atok.rb]: Why is this commented out?

Comment (ohkura) [inline Casks/atok.rb]: Oops, sorry we don't need this anymore.  It's the default uninstaller provided by the package, but it enforces to shutdown all the other apps, including terminal, and prevent us to properly finish running the brew cask uninstall command.

**Comment by miccal** [comment]
Thank you for your contribution.

**Comment by ohkura** [comment]
Thank you!

**Comment by miccal** [comment]
You're welcome.

**Pull request merged**